app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = "Ruby on Rails Tutorial Sample App"
+    page_title.empty? ? base_title : "#{page_title} | #{base_title}"
```

Comment:
1. Giải thích code line 2
2. phân biệt blank? empty? và nil? giúp chị nè 

3. tìm hiểu `I18n in rails `và áp dụng ` "Ruby on Rails Tutorial Sample App"`